experiments/run-big-bench.py

```diff
@@ -317,8 +317,7 @@ def main(
                         )
                     index.reorder(strategies=reordering_strategies)
 
-                if num_search_threads > 1:
```

Comment:
validation is supposed to be handled by the argument parser. For search, we should be using 1 thread anyways.